src/platform/packages/shared/response-ops/alerts-delete/README.md

```diff
@@ -0,0 +1,52 @@
+# @kbn/response-ops-alerts-delete
+
+This package provides components and utilities for handling the deletion of alerts by response-ops.
+
+## Storybook
+
+```sh
+> node scripts/storybook response-ops
+```
+
+## Usage
+
+### Modal
+
+The `AlertsDeleteModal` component can be used to display a confirmation modal for deleting alerts.
```

Comment:
```suggestion
The `AlertDeleteModal` component can be used to display a confirmation modal for deleting alerts.
```

I was `ctrl+f`-ing, looking for the declaration, and couldn't find it. That's how I noticed the typo 😅